From 646fbe51d925a4eac45009bda2c2fcb1b26e8a70 Mon Sep 17 00:00:00 2001
From: William Cohen <wcohen@redhat.com>
Date: Thu, 5 Apr 2012 08:33:08 -0400
Subject: [PATCH] Make autogen.sh a bit more discriminating on automake
 version

Fedora Rawhide has automake version 1.11.4 which has the substring "1.4"
in it. This would cause autogen.sh to fail. Made the grep in autogen.sh
more discriminating to avoid the incorrect matches.

Signed-off-by: William Cohen <wcohen@redhat.com>
---
 autogen.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 5148e80..40d09d0 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -21,7 +21,7 @@ AUTOHEADER=${AUTOHEADER:-autoheader}
 AUTOMAKE=${AUTOMAKE:-automake}
 AUTOCONF=${AUTOCONF:-autoconf}
 
-if $AUTOMAKE --version | grep -q 1.4
+if $AUTOMAKE --version | grep -q " 1.4$"
 	then
 	echo ""
 	echo "Automake 1.4 not supported. please set \$AUTOMAKE"
-- 
1.7.9.5

